l1-contracts/contracts/bridgehub/Bridgehub.sol

```diff
@@ -767,6 +767,19 @@ contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, Paus
         });
     }
 
+    /// @dev Registers an already deployed chain with the bridgehub
+    /// @param _chainId The chain Id of the chain
+    /// @param _hyperchain Address of the hyperchain
+    function registerAlreadyDeployedHyperchain(uint256 _chainId, address _hyperchain) external onlyOwnerOrAdmin {
```

Comment:
Some more values need to set, the stm of the chain, the baseTokenAddress/baseTokenAssetId, etc. you can copy the values from the createNewChain function: 

```
    function createNewChain(
        uint256 _chainId,
        address _stateTransitionManager,
        bytes32 _baseTokenAssetId,
        // solhint-disable-next-line no-unused-vars
        uint256 _salt,
        address _admin,
        bytes calldata _initData,
        bytes[] calldata _factoryDeps
    ) external
```